handbook/technical-overview/auth/system/service-provider-manual.md

```diff
@@ -0,0 +1,11 @@
+# Service Provider Manual
+
+[**Audience**](service-provider-manual/audience.md)
+
+[**The authentication service provision**](service-provider-manual/authentication-service-provision.md)
+
+[**Trouble-shooting**](service-provider-manual/trouble-shooting.md)
+
+[**Incidents**](service-provider-manual/incidents.md)
+
+[**Intervention by code change**](service-provider-manual/intervention-by-code-change.md)
```

Comment:
Again bolded links, is that necessary when all of them are bolded and the file contains only links?